fdbclient/ServerKnobs.cpp

```diff
@@ -365,6 +365,11 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( DD_MAXIMUM_LARGE_TEAM_CLEANUP,                       10000 ); if( randomize && BUGGIFY ) DD_MAXIMUM_LARGE_TEAM_CLEANUP = 10;
 	init( DD_LARGE_TEAM_DELAY,                                  60.0 );
 	init( DD_FIX_WRONG_REPLICAS_DELAY,                          60.0 );
+	
+	// BulkLoading
+	init( DD_BULKLOAD_FINISH_MOVE_SHARD_RETRY_MAX,                50 );
+	init( BULKLOAD_FILE_BYTES_MAX,                  1*1024*1024*1024 ); // 1GB
```

Comment:
Should we check this size limit before accepting the bulk load?

Reply:
Yes. I prefer to delay this check to when we have a file download method implemented like S3.

Reply:
ok